lib/orbit-common/cache.js

```diff
@@ -186,9 +184,9 @@ var Cache = Class.extend({
     };
 
     var performDependentOps = function() {
-      dependentOperations.forEach(function(operation) {
-        _this.transform(operation);
-      });
+      for (var i = 0, len = dependentOperations.length; i < len; ++i) {
```

Comment:
```js
Would this be any worse/better?

```js
while (dependentOperations.length) {
  _this.transform(dependentOperations.shift());
}
```

Also - does `transform` take an array? I thought it could/did?